DataFormats/FWLite/interface/Event.h

```diff
@@ -87,7 +88,8 @@ namespace fwlite {
       public:
          // NOTE: Does NOT take ownership so iFile must remain around
          // at least as long as Event
-         Event(TFile* iFile);
+         Event(TFile* iFile, bool useCache=true,
+               std::function<void (TBranch*)> baFoo=[](TBranch*){});
```

Comment:
This is BranchAccess callback, just passed to DataGetterHelper ... so an external entity can be notified about a specific access to a branch. In this case, Fireworks ... so it can adjust the state of its tree-cache.